src/Illuminate/Mail/Mailer.php

```diff
@@ -335,10 +337,48 @@ protected function parseView($view)
 	}
 
 	/**
+	 * Reset Swift Mailer SMTP transport adapter
+	 *
+	 * @return void
+	 */
+	protected function resetSwiftTransport()
+	{
+		$transport = $this->getSwiftMailerTransport();
+
+		if (!isset($transport))
+		{
+			return;
+		}
+
+		try
+		{
+			// Send RESET to restart the SMTP status and check if it's ready for running
+			if ($transport instanceof Swift_Transport_AbstractSmtpTransport)
+			{
+				$transport->reset();
+			}
+		}
+		catch ( Exception $e )
+		{
+			// In case of failure - let's try to restart it
+			try
+			{
+				$transport->stop();
+			}
+			catch ( Exception $e )
+			{
+				// Just start it then...
+			}
+
+			$transport->start();
+		}
+	}
+
+	/**
 	 * Send a Swift Message instance.
 	 *
 	 * @param  \Swift_Message  $message
-	 * @return void
+	 * @return mixed
```

Comment:
Yes, we know. We don't care about that, as I said, we're offically saying that the return value of this method cannot be relied on, so we're using `void` to indicate this.